rpa-builder/6.6/modules/ROOT/pages/toolbox-file-operations-extract-pages-from-pdf-file.adoc

```diff
@@ -0,0 +1,63 @@
+= Extract Pages from PDF File
+
+Extract pages from a PDF file, customize the output, and create a new PDF file with the configured settings.
+
+== Properties
+
+* *File Settings*
+* *Create Target PDF if it doesn't exist*
++ 
+Select this option to create the PDF file if the specified output file name doesn't exist. Otherwise, the action step fails if the file doesn't exist. 
+* *Extraction Page Range* 
++
+The pages to extract from the input PDF. Use commas (`,`) to separate multiple ranges and `EOF` to specify the end of the file. This action step reads the pages in the specified order. For example, the value`1-5, EOF-6` outputs a file that contains pages from 1 to 5 in the original order and the remaining pages in reverse order, from last to 6.
+
+* *Input File Directory Path* 
++
+The path to the directory where the input file is. 
+* *Input File is Password Protected* 
++
+Select this option if the input file is password protected and specify the password in the Wizard. 
+* *Input File Name* 
++
+The name of the input file. 

```

Comment:
I think Diane prefers leaving off the article for incomplete sentences (noun descriptions):

Name of the input file.